Jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline
       // ENVs for check-compliance
       GH_TOKEN = credentials('nordicbuilder-compliance-token') // This token is used to by check_compliance to comment on PRs and use checks
       GH_USERNAME = "NordicBuilder"
-      COMPLIANCE_ARGS = "-r NordicPlayground/nrfxlib"
+      COMPLIANCE_ARGS = "-r nrfconnect/nrfxlib"
```

Comment:
Should this have been `nrfconnect/sdk-nrfxlib` ?

Reply:
yes, thank you!